libfuse/debug.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libfuse
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/keybase/client/go/logger"
+)
+
+// MakeFuseDebugFn returns a function that logs its argument to the
+// given log, suitable to assign to fuse.Debug.
+func MakeFuseDebugFn(
+	log logger.Logger, superVerbose bool) func(msg interface{}) {
+	return func(msg interface{}) {
+		str := fmt.Sprintf("%s", msg)
+		// If superVerbose is not set, filter out Statfs and
```

Comment:
My plan is to use https://github.com/uber-go/zap for structured logging, I've been just waiting for it to mature. That'll hopefully give more power for log filtering. There is a context that could be passed to the logging, that just means changing the API, so hopefully that'll be done in one fell swoop.